azerothcore/azerothcore-wotlk
# Update world.playercreateinfo_action to line up with Wrath Classic.

## Changes Proposed:
-  Updates positioning of many spells in the action bars of a newly created character to match up with Wrath Classic PTR.

## Issues Addressed:

- Closes unreported issue.

## SOURCE:

Wrath Classic PTR

Videos found here: [1](https://www.youtube.com/watch?v=rwsr_HWOOSg), [2](https://www.youtube.com/watch?v=7E3YLQJfHZE), [3](https://www.youtube.com/watch?v=86L6iz_rCrg), [4](https://www.youtube.com/watch?v=_Gn7WqbGbDM)

## Tests Performed:

- Tested a few as I went. Will need to check on the Wrath PTR what spells are in the default stealth bar for rogues, as well as the different transformations for druids.


## How to Test the Changes:


1. Run the query
2. Log into a given race/class combination to make sure they have the correctly positioned spells according to the video source.

## Known Issues and TODO List:


- [ ] Check rogue stealth bar
- [ ] Check druid transformation bars


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
